roles/etcd/tasks/backup/backup.yml

```diff
@@ -49,10 +49,17 @@
   - l_etcd_selinux_labels.rc == 0
   - "'svirt_sandbox_file_t' not in l_etcd_selinux_labels.stdout"
 
-- name: Generate etcd backup
+- name: Generate etcd backup (legacy)
```

Comment:
In what context will this legacy backup be used from the 3.10+ branch?  Is this still needed?

Reply:
It’s needed for 3.9->3.10, we won’t have a static pod until after the control plane updates.